explanations/dispatchers.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>See previous comment -- first explain what is happening, then say that consumption will of course take on different values and we'll want to vectorize over it.</p><p> </p><p>It is not fully obvious to me why we have spacemap applying sparse choices but need to do a manual productmap for continuous choices. At least seems to merit a brief explanation!</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/lcm/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>